sys/vm/vm_mmap.c

```diff
@@ -797,8 +806,7 @@ kern_mmap_req(struct thread *td, struct mmap_req *mrp)
 			return (error);
 		flags &= ~MAP_EXCL;
 		flags |= MAP_FIXED;
-		mrp->mr_source_cap = mmap_retcap(td,
-		    addr + pageoff, mrp);
+		mrp->mr_source_cap = mmap_retcap(td, addr + pageoff, mrp);
```

Comment:
Maybe deserves to be in a separate commit if you want to make that whitespace change?